RadarSDK/Include/Radar.h

```diff
@@ -492,6 +492,14 @@ typedef void (^_Nonnull RadarLogConversionCompletionHandler)(RadarStatus status,
  */
 + (void)getVerifiedLocationToken:(RadarTrackVerifiedCompletionHandler _Nullable)completionHandler NS_SWIFT_NAME(getVerifiedLocationToken(completionHandler:));
 
+/**
+ Optionally sets the user's expected country and state for jurisdiction checks.
+ 
+ @param countryCode The user's expected two-letter country code.
+ @param stateCode The user's expected two-letter state code.
+ */
++ (void)setExpectedJurisdictionWithCountryCode:(NSString *_Nullable)countryCode stateCode:(NSString *_Nullable)stateCode NS_SWIFT_NAME(setExpectedJurisdiction(countryCode:stateCode:));
```

Comment:
If you scroll all the way to the right, it will be `setExpectedJurisdiction()` in Swift. `setExpectedJurisdictionWithCountryCode:stateCode:` is just an Obj-C convenion